java/src/org/openqa/selenium/firefox/FirefoxOptions.java

```diff
@@ -66,6 +66,7 @@ public FirefoxOptions() {
     // will enable it.
     // https://fxdx.dev/deprecating-cdp-support-in-firefox-embracing-the-future-with-webdriver-bidi/.
     addPreference("remote.active-protocols", 3);
```

Comment:
@pujagani do you know why this preference is still set to `3` for Java, which basically means that CDP is enabled in Firefox? I thought that with https://github.com/SeleniumHQ/selenium/issues/11736 all the CDP stuff was removed. Maybe it was forgotten?

Reply:
There was pending PR for this. Let me go sort that out. Thank you!